tests/Unit/MaterialControllerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Http\Controllers\MaterialController;
+use App\Http\Requests\Material\StoreRequest;
+use App\Http\Requests\Material\UpdateRequest;
+use App\UseCases\Material\DestroyUseCase;
+use App\UseCases\Material\StoreUseCase;
+use App\UseCases\Material\UpdateUseCase;
+use Database\Seeders\CategoriesTableSeeder;
+use Database\Seeders\MaterialsTableSeeder;
+use Database\Seeders\UsersTableSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MaterialControllerTest extends TestCase
+{
+    use RefreshDatabase;
+    use WithFaker;
+
+    public function setup(): void
+    {
+        parent::setUp();
+        $this->seed(UsersTableSeeder::class);
+        $this->seed(CategoriesTableSeeder::class);
+        $this->seed(MaterialsTableSeeder::class);
+    }
```

Comment:
(これは言っていなかったことなので、すみません :man-bowing: )
テストケースでsetupを入れた場合は、セットでtearDownも入れておいてください

```suggestion
    }
    
    public function tearDown(): void
    {
           parent::tearDown();
    }
```